consensus/polybft/polybft.go

```diff
@@ -225,42 +214,42 @@ func GenesisPostHookFactory(cfg *chain.Chain, engineName string) func(txn *state
 					// initialize ChildERC20PredicateAccessList SC
 					if err = initERC20ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, false, chainIDBig,
-						"ChildERC20PredicateAccessList"); err != nil {
+						"InternalERC20Predicate"); err != nil {
```

Comment:
Change on other places as well.
```suggestion
						"InternalERC20ACLPredicate"); err != nil {
```